context/app/static/js/components/home-revision/AssayTypeBarChartContainer/style.js

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components';
+
+const Flex = styled.div`
+  display: flex;
+`;
+
+const ChartWrapper = styled.div`
+  width: 1000px;
+  height: 800px;
+`;
```

Comment:
I guess this is not a comment about this PR specifically, but is there no resizing here?  It seems like the legend does react a bit to resizing.

Reply:
The sizing here is just temporary until I fit it into the home page.